vector/src/main/java/im/vector/app/features/MainActivity.kt

```diff
@@ -131,6 +141,8 @@ class MainActivity : VectorBaseActivity<ActivityMainBinding>(), UnlockedActivity
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        shortcutsHandler.updateShortcutsWithPreviousIntent()
```

Comment:
eventually we may want to introduce an app domain migrator for these sorts of changes (could also be used for updating and removing shared prefs etc)

Reply:
yes, it was requested by other devs too.